aiven/client/connection_info/common.py

```diff
@@ -19,7 +20,7 @@ class Store(Enum):
     write = object()
     skip = object()
 
-    def handle(self, getter, path):
+    def handle(self, getter: Callable[..., str], path: str) -> None:
```

Comment:
Looks like it should be `Callable[[], str]` instead. The `...` means "any arguments" but the getter is used without any arguments.